### PR TITLE
Issue #389- fixed Gem dependencies

### PR DESCRIPTION
**Pull request by wiseemily88**
Added Ruby version to Gemfile, and  updated the ReadMe for those using a RVM specifying Ruby version and any potential issues with an outdated Bundler on your local machine. PR submitted with the help from @izaxveta and @erikaannesmith

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -20,22 +20,26 @@ The Rails Girls Summer of Code website is built using [Jekyll](https://jekyllrb.
...
+
 ```
 gem install bundler
 bundle install
 jekyll serve --watch
 
 ```
 
+#### Note:Ruby Dependencies
+Both .ruby-version and Gemfile handle the Ruby version (2.1.5) for this application. Users with or without a RVM (Ruby Versioning Manager) should be able to interact and contribute to the code base. If running an older local Ruby version on your machine, you may need to `bundle update` to reset Gemfile.lock.
```
Comment (alicetragedy) [inline CONTRIBUTING.md]: two comments about this section:  
- I don't think it's necessary to specify the exact Ruby version here, you just remove the `(2.1.5)` — that way we could avoid the README becoming accidentally outdated the next time we update our Ruby version. What do you think?
- I'd suggest to make the bit about Ruby versioning more general, something along the lines of:
"User who don't use a tool to manage Ruby versions (such as [RVM](https://rvm.io/) or [rbenv](https://github.com/rbenv/rbenv)) should be able to [...]". This way we avoid to specifically show preference about a single tool.

File: Gemfile [hunk 2]
```diff
@@ -1,6 +1,7 @@
 source 'https://rubygems.org'
 
-gem 'jekyll', '~> 3.1.6'
-gem 'github-pages', '84'
+gem 'jekyll'
+gem 'github-pages'
 gem 'geocoder'
 gem 'lolize', :require => 'lolize/auto'
+ruby '~> 2.1.5'
```
Comment (alicetragedy) [inline Gemfile]: A suggestion: replace this line with `ruby File.read(".ruby-version")`.
This way, we'd only need to update the ruby version in the `.ruby-version` file, and the Gemfile would follow. :)

Comment (alicetragedy) [reply Gemfile]: ah and also:
while I don't actually know the reason for it (or *if* there is a reason for it) the convention in a Gemfile is usually to put the Ruby version right under the `source 'https://rubygems.org'` line. Could you move it up there?

**Review by alicetragedy — CHANGES_REQUESTED** [review]
Hi @wiseemily88, thank you so much for looking into this issue and providing a fix! It's awesome, and I'm so grateful for these changes you've made 🙌 🎉  
I've added a couple of minor suggestions of things that would need changing before we can merge this. Please let me know if you have any questions about my comments or if anything is unclear!

**Comment by alicetragedy** [comment]
hi @wiseemily88 👋 
I'd love to merge your changes and implement those improvements you suggested! In order to do that, I'd need you to review my suggestions in the PR review above. Please me know if you have any questions.
If you think you won't have time, or if you are not interested in doing this anymore, please leave a comment in here — that way I can cherry-pick your commits (or merge your PR and make those additional changes myself, if that's ok with you). 🎉

**Comment by wiseemily88** [comment]
@alicetragedy Sorry this took so long to address!! look forward to contributing more and at a quicker pace in the future :)

**Review by alicetragedy — APPROVED** [review]

**Comment by alicetragedy** [comment]
This is so great @wiseemily88 — thank you so much! So happy to be able to merge this and implement your awesome changes into our project 🎉 
We'd love to have you contribute again, I'm always glad to see contributors helping out. 🙌 
Congrats on your first contribution to the RGSoC project!

**Pull request merged**